### ISSUE TITLE
print function in string template bug

### ISSUE THREAD

**Issue by LaysDragon** [mlogjs/mlogjs#115]
version: 0.4.5
```ts
function switch_test(value: number) {
    let whatEver = 'whatEver';
    switch (value) {
        case 0:
            whatEver =  'case0';
            break;
        case 1:
            whatEver =  'case1';
            break;
        case 2:
            whatEver =  'case2';
            break;
    }
    return whatEver;
}

print('case0: ',switch_test(0),' case1: ',switch_test(1),' case2: ',switch_test(2))
printFlush();
```

compile
```
set value:1:21 0
set &rswitch_test:1:0 3
jump 17 always
set value:1:21 1
set &rswitch_test:1:0 6
jump 17 always
set value:1:21 2
set &rswitch_test:1:0 9
jump 17 always
print "case0: "
print &fswitch_test:1:0
print " case1: "
print &fswitch_test:1:0
print " case2: "
print &fswitch_test:1:0
printflush message1
end
set whatEver:2:8 "whatEver"
jump 22 strictEqual value:1:21 0
jump 24 strictEqual value:1:21 1
jump 26 strictEqual value:1:21 2
jump 27 always
set whatEver:2:8 "case0"
jump 27 always
set whatEver:2:8 "case1"
jump 27 always
set whatEver:2:8 "case2"
set &fswitch_test:1:0 whatEver:2:8
set @counter &rswitch_test:1:0

```

result:
![image](https://user-images.githubusercontent.com/10185394/199128239-b4e9cf0e-4e10-4451-bf5b-a6b495e75a7a.png)

expected: case0:case0 cacse1:case1 case2:case2

**Comment by JeanJPNM** [comment]
The return value of the function is getting overridden

**Comment by JeanJPNM** [comment]
Fixed in version 0.4.6
